google_drive/google_calendar.py

```diff
@@ -105,9 +97,18 @@ def studio_submit(self, submissions, suffix=''):  # pylint: disable=unused-argum
         """
         Change the settings for this XBlock given by the Studio user
         """
-        self.display_name = submissions['display_name']
-        self.calendar_id = submissions['calendar_id']
-        self.default_view = submissions['default_view']
+        if not isinstance(submissions, dict):
+            LOG.error("submissions object from Studio is not a dict - %s", unicode(submissions))
```

Comment:
unicode(submissions) will itself fail if it's a string with non-ascii characters.  The best option for how to log unknown data is `"not a dict - %r" % (submissions,)`.